WeakAurasOptions/ForAllIndentsAndPurposes.lua

```diff
@@ -974,16 +974,21 @@ if not IndentationLib.revision or revision > IndentationLib.revision then
 
     -- returns the padded code, and true if modified, false if unmodified
     local linebreak = stringbyte("\n")
+    local space = stringbyte(" ")
     function lib.padWithLinebreaks(code)
-        local len = stringlen(code)
-        if stringbyte(code, len) == linebreak then
-            if stringbyte(code, len - 1) == linebreak then
-                return code, false
+        local len = stringlen(code) + 1
+        local breakfound = 0
+        while true do
+            len = len - 1
+            local byte = stringbyte(code, len)
```

Comment:
This might cause issues if the code is < 2 bytes long. I’m not sure how stringbyte handles inputs < 1.

Reply:
A simple fix would be to check if the string is extremely short, and handle that as a special case.

Reply:
it's working as intended on an empty custom code editbox